Libplanet/Net/Swarm.cs

```diff
@@ -1421,8 +1421,7 @@ private void BroadcastTxIds(IEnumerable<TxId> txIds)
                     }
 
                 default:
-                    Trace.Fail($"Can't handle message. [{message}]");
-                    break;
+                    throw new InvalidMessageException($"Can't handle message. [{message}]");
```

Comment:
A trivial style suggestion:

```suggestion
                    throw new InvalidMessageException($"Can't handle message: {message}");
```

Also it would be better if `InvalidMessageException` has a property like `InvalidMessageException.ReceivedMessage`.